tokens/functional/border/light.json5

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This isn't a blocking comment, but for focus styles specifically we use `outline` not `border` for most cases. I probably wasn't very clear about that, sorry! We also are somewhat flexible about the value as we tweak it for specific cases: https://github.com/primer/css/blob/main/src/support/mixins/misc.scss#L29

I think this PR is still needed though for other border tokens?

Reply:
Just saw in the issue you mention outline is similar enough to border.. that does make sense, but we'll also want tokens for `outline-offset` which unfortunately is not part of the outline shorthand. So just something to think about (we probably want all our focus style tokens to have the same property in the name)

Reply:
Ahh, I did not know about `outline-offset`. However it seems [the offset](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-offset) is not part of the [outline shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/outline).

So I would suggest the offset should be defined as a dimension/size token.

Reply:
Changed the top level name to `outline` and added the `offset`. However it is currently not being complied because we have not moved the sizes, etc. over to the new build system.

Another question that is related to the naming is if the offset should be:
- `outline-focus-offset`
- `offset-outline-focus`
- `size-outline-offset-focus`
- ...

But this can be fixed after this is merged.

Reply:
Yeah, we can fix this later once we allow camelCase in the new convention. I think it would be:

```
ui-color-outline-focus 
ui-size-outline-focus
ui-size-outlineOffset-focus
```

Reply:
We would also need something like:
```sh
--ui-border-outline-focus: [color] [style] [width];
# or 
--ui-outline-focus: [color] [style] [width];
```